smartcontractkit/external-adapters-js
# add per second and per minute rate limit environment variables

## Closes 22258

## Description

Allow user to set either `RATE_LIMIT_PER_SECOND` or `RATE_LIMIT_PER_MINUTE` to override the `RATE_LIMIT_CAPACITY` environment variable

## Changes

- Add `RATE_LIMIT_PER_SECOND` or `RATE_LIMIT_PER_MINUTE` variables and let them override `RATE_LIMIT_CAPACITY` 



## Steps to Test

1) Set `RATE_LIMIT_CAPACITY_SECOND` and `RATE_LIMIT_CAPACITY_MINUTE` environment variables
2) Run alphaavantage adapter
3) Send 6 unique requests using Postman within one minute
4) Verify that he following are true
  - Adapter returns 200 response
  - Error message below does not show up
 ```
 {"level":"warn","time":1637775053323,"pid":28,"hostname":"alphavantage-b7bcfdd76-8n6t6","msg":"External Adapter backing off. Provider's limit of 4.5 requests per minute reached. 5 requests sent in the last minute."}
```

## Quality Assurance
